connection.go

```diff
@@ -524,7 +524,10 @@ func (c *Connection) Reply(ctx context.Context, opErr error) error {
 
 	// Error logging
 	if c.shouldLogError(op, opErr) {
-		c.errorLogger.Printf("%T error: %v", op, opErr)
+		msg := fmt.Sprintf(
```

Comment:
Any reason for splitting this into a separate fmt.Sprintf call and using Println instead of sticking to Printf? This could just be:

```go
c.errorLogger.Printf("Op 0x%08x %T] -> Error: %q", fuseID, op, opErr)
```

…right?

Reply:
Hmm SG.